### ISSUE TITLE
Make website accessible for ALL (major) screen readers.

### ISSUE THREAD

**Issue by vilihegyi** [vilihegyi/securo-webapp#1]
Website should be accessible with JAWS, NV Access, (iOS) VoiceOver and GoogleTalkBack screen readers.
